dev-python/slimit/slimit-0.8.1-r1.ebuild

```diff
@@ -25,6 +25,14 @@ DEPEND="${RDEPEND}
 
 PATCHES=( "${FILESDIR}/${P}-fix-python3.patch" )
 
+python_compile() {
+	distutils-r1_python_compile
+
+	rm -f "${BUILD_DIR}"/lib/slimit/*tab.py
+
+	PYTHONPATH="${BUILD_DIR}/lib" "${EPYTHON}" -B -c 'import slimit;slimit.minify("")' || die
```

Comment:
Why do you need to override PYTHONPATH? Also please add a comment explaining what this does.